app/src/common/urls.js

```diff
@@ -127,6 +127,8 @@ export const URLS = {
 
   organizationList: (preferencesObj = {}) =>
     `${urlCommonBase}organizations${getQueryParams(preferencesObj)}`,
+  organizationProject: (organizationId) =>
```

Comment:
Can we use `organizationProjects` URL instead?
The query params object will be just empty.